tests/utils/services.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetServiceInNamespaceOrDir(c *kubernetes.Clientset, ns, name string) *v1.Service {
+	svc, err := GetServiceInNamespace(c, ns, name)
+	if err != nil {
+		glog.Fatal(err)
+	}
+	return svc
+}
+
+func GetServiceInNamespace(c *kubernetes.Clientset, ns, name string) (*v1.Service, error) {
+	svc, err := c.CoreV1().Services(ns).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to get service \"%s/%s\"", ns, name)
+	}
+	return svc, nil
+}
+
+func GetServicesInNamespaceByLabel(c *kubernetes.Clientset, ns, labelSelector string) (*v1.ServiceList, error) {
+	svcs, err := c.CoreV1().Services(ns).List(metav1.ListOptions{
+		LabelSelector: labelSelector,
+	})
+	if err != nil {
+		return nil, errors.Wrapf(err, "Failed to get services for label selector %q", labelSelector)
+	}
+	return svcs, nil
+}
+
+func GetServicePortByName(svc *v1.Service, name string) int {
```

Comment:
The func name and the return being an `int` seem slightly contradictory to me. I was expecting a string to be returned since _Name_ is in the func name.